### ISSUE TITLE
Proposal: Show the power in a demo project online

### ISSUE THREAD

**Issue by pmusolino** [calebkleveter/UIWebKit#7]
Cool project! I want to suggest you to open a demo website realized with this framework, where how want to use UIWebKit can explore its power and its features.

**Comment by calebkleveter** [comment]
Yes. I should do this sometime. I am currently tangled up with a bunch of other stuff, so I don't  have much time, but I will do this as soon as possible.
